src/components/Composer/Editor/index.tsx

```diff
@@ -47,4 +89,9 @@ const Editor: FC = () => {
   );
 };
 
+Editor.defaultProps = {
+  placeholder: '',
+  defaultContent: 'Hello world'
```

Comment:
Shouldn't this also be `Hello world` check the comment editor to see what happens 😛

Reply:
Right, forgot to remove this string. I set it to empty string.